arangosh/Dump/DumpFeature.cpp

```diff
@@ -954,7 +959,13 @@ int DumpFeature::runClusterDump(std::string& errorMsg) {
 
       beginEncryption(fd);
 
-      std::string const collectionInfo = collection.toJson();
```

Comment:
Can we replace it with a simple `AttributeExcludeHandler`
that implements:
shouldExclude(key, nesting) => if (key == "shadowCollections" && nesting == 1) {return false;}else { return true;}

Reply:
Alternative:

Builder b = VPackCollection::remove(slice, {"parameters", "shadowCollections"}) should do the trick

Reply:
Adressed in https://github.com/arangodb/arangodb/pull/5065/commits/986a5270137d784b14fbd4df3204228dcf12af6e